src/cli.rs

```diff
@@ -36,61 +42,56 @@ pub(crate) struct Args {
     pub render: Option<PathBuf>,
 
     /// Override the operating system
-    #[clap(
+    #[arg(
         short = 'p',
         long = "platform",
         action = ArgAction::Append,
-        possible_values = ["linux", "macos", "windows", "sunos", "osx", "android", "freebsd", "netbsd", "openbsd"],
     )]
     pub platforms: Option<Vec<PlatformType>>,
 
     /// Override the language
-    #[clap(short = 'L', long = "language")]
+    #[arg(short = 'L', long = "language")]
     pub language: Option<String>,
 
     /// Update the local cache
-    #[clap(short = 'u', long = "update")]
+    #[arg(short = 'u', long = "update")]
     pub update: bool,
 
     /// If auto update is configured, disable it for this run
-    #[clap(long = "no-auto-update", requires = "command_or_file")]
+    #[arg(long = "no-auto-update", requires = "command_or_file")]
     pub no_auto_update: bool,
 
     /// Clear the local cache
-    #[clap(short = 'c', long = "clear-cache")]
+    #[arg(short = 'c', long = "clear-cache")]
     pub clear_cache: bool,
 
     /// Use a pager to page output
-    #[clap(long = "pager", requires = "command_or_file")]
+    #[arg(long = "pager", requires = "command_or_file")]
     pub pager: bool,
 
     /// Display the raw markdown instead of rendering it
-    #[clap(short = 'r', long = "--raw", requires = "command_or_file")]
+    #[arg(short = 'r', long = "raw", requires = "command_or_file")]
     pub raw: bool,
 
     /// Suppress informational messages
-    #[clap(short = 'q', long = "quiet")]
+    #[arg(short = 'q', long = "quiet")]
     pub quiet: bool,
 
     /// Show file and directory paths used by tealdeer
-    #[clap(long = "show-paths")]
+    #[arg(long = "show-paths")]
     pub show_paths: bool,
 
     /// Create a basic config
-    #[clap(long = "seed-config")]
+    #[arg(long = "seed-config")]
     pub seed_config: bool,
 
     /// Control whether to use color
-    #[clap(
-        long = "color",
-        value_name = "WHEN",
-        possible_values = ["always", "auto", "never"]
-    )]
+    #[arg(long = "color", value_name = "WHEN")]
     pub color: Option<ColorOptions>,
 
     /// Print the version
     // Note: We override the version flag because clap uses `-V` by default,
     // while TLDR specification requires `-v` to be used.
-    #[clap(short = 'v', long = "version")]
-    pub version: bool,
+    #[arg(short = 'v', long = "version", action = ArgAction::Version)]
+    pub version: Option<bool>,
```

Comment:
This is a bit weird, I would have thought that this type should only have two values, not three as `Option<bool>` - but `bool` and `Option<()>` both fail, so I guess this is the way to go?

Reply:
Thanks to GitHub code search, I found many results with `Option<bool>`, but also some results that used `()`. And that actually works 🙂